py/private/py_image_layer.bzl

```diff
@@ -0,0 +1,138 @@
+"py_image_layer"
+
+load("@aspect_bazel_lib//lib:tar.bzl", "mtree_spec", "tar")
+
+default_layer_groups = {
+    # match *only* external pip like repositories that contain the string "pip_deps"
+    "packages": "\\.runfiles/pip_deps.*",
```

Comment:
what if the `pip_install` was named something else? Even in this repo we use `pypi` as the name rather than `pip` - so if you had an example you'd notice.

Maybe it should be a placeholder that can be separately filled with an attribute like `deps_repo_name`

Reply:
hmm, i should probably capture, `site-packages/`, similar to what i did for `.runfiles/python`

Reply:
okay i replaced it with `\\.runfiles/.*/site-packages` which will match regardless of repo name.